### PR TITLE
zong0jie

### PR DESCRIPTION
**Pull request by zong0jie**
This work is very good and I want to use it in my project. However I find that here may have error when I want to copy or move files in hdfs.
So I modify the HadoopFileSystem.java and make it works.
I would like to know whether the work is useful.

[Review on Reviewable](https://reviewable.io/reviews/damiencarol/jsr203-hadoop/20)

### CODE REVIEW

File: src/test/java/hdfs/jsr203/TestFileSystem.java [hunk 1]
```diff
@@ -310,4 +311,29 @@ public void getRootDirectories() throws IOException {
...
+    
+    @Test
+    public void testCopyeAndMoveFiles() throws IOException {
+        URI uriSrc = clusterUri.resolve("/tmp/testSrcFile");
+        Path pathSrc = Paths.get(uriSrc);
+        
+        URI uriDstCp = clusterUri.resolve("/tmp/testDstCopyFile");
+        Path pathDstCp = Paths.get(uriDstCp);
+        
+        BufferedWriter br = Files.newBufferedWriter(pathSrc);
```
Comment (damiencarol) [inline src/test/java/hdfs/jsr203/TestFileSystem.java]: It seems Travis break here :/
Could you investigate?
```
[ERROR] /home/travis/build/damiencarol/jsr203-hadoop/src/test/java/hdfs/jsr203/TestFileSystem.java:[323,34] method newBufferedWriter in class java.nio.file.Files cannot be applied to given types;

  required: java.nio.file.Path,java.nio.charset.Charset,java.nio.file.OpenOption[]

  found: java.nio.file.Path

  reason: actual and formal argument lists differ in length

[INFO] 1 error
```

**Comment by damiencarol** [comment]
@zong0jie thanks a lot for your interest and I'm glad you want to be part of this project :)
I'm analyzing your PR. Will make FB soon.

File: src/test/java/hdfs/jsr203/TestFileSystem.java (hunk 1 above)
Comment (zong0jie) [inline src/test/java/hdfs/jsr203/TestFileSystem.java]: Maybe I use the jdk 1.8 Jdk1.7 environment. On my computer the BufferedWriter works. 
I change it to OutputStream and it's ok.

Comment (zong0jie) [inline src/test/java/hdfs/jsr203/TestFileSystem.java]: Maybe I use jdk 1.8 Jdk1.7 environment. On my computer the BufferedWriter works. 
I change it to OutputStream and it's ok.

**Comment by damiencarol** [comment]
Thanks.

**Comment by zong0jie** [comment]
Thank you for the merge. If you have  some other plan which you may not have enough time, maybe you can let me know, I think I can do something for you.
It's the first time I join an open source project, so please point my error at anytime.

**Pull request merged**